apps/root/src/pages/strategy-guardian-detail/investment-data/components/wallet-breakdown/index.tsx

```diff
@@ -0,0 +1,193 @@
+import Address from '@common/components/address';
+import ComposedTokenIcon from '@common/components/composed-token-icon';
+import { formatUsdAmount, isSameToken } from '@common/utils/currency';
+import { BalanceToken } from '@hooks/useMergedTokensBalances';
+import { TokenNetworksTooltipTitle } from '@pages/home/components/token-icon-multichain';
+import { DisplayStrategy } from 'common-types';
+import React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import styled from 'styled-components';
+import {
+  Accordion,
+  AccordionSummary,
+  Typography,
+  AccordionDetails,
+  StyledBodySmallLabelTypography,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Tooltip,
+  colors,
+} from 'ui-library';
+
+interface WalletBreakdownProps {
+  strategy: DisplayStrategy;
+}
+
+const StyledCell = styled(TableCell)`
+  padding: 0;
+  padding-left: 0px !important;
+  padding-right: 0px !important;
+`;
+
+const StyledRow = styled(TableRow)<{ $isLast: boolean; $isFirst: boolean }>`
+  background-color: transparent !important;
+  ${({
+    theme: {
+      palette: { mode },
+      spacing,
+    },
+    $isFirst,
+    $isLast,
+  }) => `
+    ${StyledCell} {
+      padding-top: ${$isFirst ? '0px' : spacing(2)};
+      padding-bottom: ${$isLast ? '0px' : spacing(2)};
+      ${!$isLast && `border-bottom: 1.5px solid ${colors[mode].border.border1};`}
+      border-radius: 0px;
+    }
+  `}
+`;
+
+const WalletBreakdown = ({ strategy }: WalletBreakdownProps) => {
+  const intl = useIntl();
+
+  return (
+    <Accordion disableGutters defaultExpanded sx={{ padding: ({ spacing }) => spacing(4) }}>
+      <AccordionSummary>
+        <Typography variant="h6Bold">
+          <FormattedMessage defaultMessage="Breakdown" description="strategy-detail.vault-investment-data.breakdown" />
```

Comment:
To keep consistency, we should pre-pend the application (earn) to this descriptions